lib/diagnostic.js

```diff
@@ -39,20 +52,33 @@ let owner = null;
 // Methods: vocalize
 
 // Replace null with your answer
-let pet = null;
+let pet = {
+  name: 'Jyo',
+  species: 'cat',
+  breed: 'domestic shorthair',
+  vocalization: 'meow',
+  vocalize: function () {}
```

Comment:
You want to make sure your method returns something, in this case the inside should look something like this: `return this.vocalization`